.github/workflows/deploy-contract.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy Contract
+on:
+  pull_request:
+    branches:
+      - "development"
+
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@master
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/development'"
+
+  install-deps:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2.1.2
+      - name: install runsim
+        run: |
+          export GO111MODULE="on" && go get github.com/cosmos/tools/cmd/runsim@v1.0.0
+      - uses: actions/cache@v2.1.1
+        with:
+          path: ~/go/bin
+          key: ${{ runner.os }}-go-runsim-binary
```

Comment:
not sure why we need to install runsim? seems unrelated

Reply:
haha woops i was using that as a reference but accidentally pushed!